server_test.go

```diff
@@ -1456,7 +1461,7 @@ func TestCompressHandler(t *testing.T) {
 	hh(&ctx)
 	s = ctx.Response.String()
 	br = bufio.NewReader(bytes.NewBufferString(s))
-	if err := resp.Read(br); err != nil {
+	if err = resp.Read(br); err != nil {
```

Comment:
Sorry if I'm being pedantic but please revert this change. It's always better to have variables scoped as tightly as possible.

Reply:
The reason this was done is because it's actually a golinter warning about shadowing.